arch/arm64/configs/aosp_loire_suzu_defconfig

```diff
@@ -708,3 +707,34 @@ CONFIG_LOGCAT_SIZE=256
 
 # ENABLE MEMNOTIFY DRIVER
 CONFIG_MEMNOTIFY=y
+
+# Enable SELINUX
+CONFIG_AUDIT=y
+# CONFIG_AUDIT_LOGINUID_IMMUTABLE is not set
+CONFIG_NETWORK_SECMARK=y
+# CONFIG_NF_CONNTRACK_SECMARK is not set
+# CONFIG_NETFILTER_XT_TARGET_AUDIT is not set
+# CONFIG_NETFILTER_XT_TARGET_SECMARK is not set
+CONFIG_SECURITYFS=y
+CONFIG_SECURITY_PATH=y
+CONFIG_LSM_MMAP_MIN_ADDR=65536
+CONFIG_SECURITY_SELINUX=y
+CONFIG_SECURITY_SELINUX_BOOTPARAM=y
+CONFIG_SECURITY_SELINUX_BOOTPARAM_VALUE=1
+# CONFIG_SECURITY_SELINUX_DISABLE is not set
+CONFIG_SECURITY_SELINUX_DEVELOP=y
+CONFIG_SECURITY_SELINUX_AVC_STATS=y
+CONFIG_SECURITY_SELINUX_CHECKREQPROT_VALUE=1
+# CONFIG_SECURITY_SELINUX_POLICYDB_VERSION_MAX is not set
+CONFIG_SECURITY_APPARMOR=y
```

Comment:
Out of curiosity, why apparmor is enabled? AFAIK it is not needed with SELinux?

Reply:
It was needed to be enabled before you were able to enable SELinux from menuconfig

Reply:
Ah, that is not needed. My bad, it was audit which was not enabled and that is why selinux was missing from menuconfig, will fix.

Reply:
Right. Without audit in config SELinux did not get enabled even with manually built kernel. Not sure why, but it apparently has to be always enabled. :)